### DISCUSSION TITLE
从 `unocss` 中导入 `defineConfig` 函数 `theme` 属性为什么是 `any` ？

### DISCUSSION THREAD

**Discussion by ktlshya** [unocss/unocss#3553] [Q&A]
![image](https://github.com/unocss/unocss/assets/21071580/b81e1e5f-148e-4b2e-a921-07ebdc5bd271)

**Comment by zyyv** [comment]
Uno is essentially a parsing engine and does not provide additional `Theme` capabilities. Theme comes from `presetMini` / `presetUno`.

You need to manually import the `Theme` type.

```ts
import type { Theme } from '@unocss/preset-mini'

export default defineConfig<Theme>({
  // xxx
})
```
